File: app/spring-webflux/src/main/java/com/jtlapp/jvmvsjs/springwebflux/HomeController.java
package com.jtlapp.jvmvsjs.springwebflux;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HomeController {

	@RequestMapping("/")
	String home() {
		return "Running spring-webflux\n";
	}
}

File: app/spring-webflux-kotlin/src/main/kotlin/com/jtlapp/jvmvsjs/springwebfluxkotlin/HomeController.kt
package com.jtlapp.jvmvsjs.springwebfluxkotlin

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HomeController {

	@RequestMapping("/")
	fun home(): String {
		return "Running spring-webflux-kotlin\n"
	}
}
